pinot-common/src/main/java/com/linkedin/pinot/common/utils/primitive/MutableLongValue.java

```diff
@@ -96,4 +96,28 @@ public boolean equals(Object o) {
   public int hashCode() {
     return EqualityUtils.hashCodeOf(value);
   }
+
+  /**
+   * Serializer for instance of the class.
+   * Serialized byte-array can be de-serialized by {@ref fromBytes}
+   *
+   * @return Serialized byte array for the instance.
+   */
+  public byte[] toBytes() {
+    ByteBuffer byteBuffer = ByteBuffer.allocate(Long.SIZE >> 3);
+    byteBuffer.putLong(value);
+    return byteBuffer.array();
```

Comment:
Does array() create separate object or shares internal representation ? 
Guava's Longs.toByteArray is another option to convert directly to array (one-liner). 
Not recommending change if we are not creating extra objects.
If you change, make sure to use same mechanism in deserializer